FueledUtils/GradientView.swift

```diff
@@ -0,0 +1,96 @@
+import UIKit
+
+public final class GradientView: UIView {
+	var gradientInfo: [(color: UIColor, location: CGFloat)] = [] {
+		didSet {
+			self.setNeedsDisplay()
+		}
+	}
+
+	@IBInspectable var isVertical: Bool = true {
```

Comment:
Do you really need this var ? It does not seem like you even read it in the class.

Reply:
@notbenoit whoops, just forgot to implement it. I've tweaked the code in `draw(_:)` to make use of it